random.menu.rb

```diff
@@ -0,0 +1,56 @@
+# establish 3 arrays with 10 items each.  1st array is of adjectives, 2nd array of cooking styles, 3rd array of foods.
+
+adjectives = [
+  "hot",
+  "blue",
+  "warm",
+  "tempura",
+  "spicy",
+  "creamy",
+  "silky",
+  "sugared",
+  "avocado",
+  "salted"
+]
+
+styles = [
+  "wok-fried",
+  "pan-seared",
+  "flash-frozen",
+  "baked",
+  "tossed",
+  "grated",
+  "au gratin",
+  "riced",
+  "julienned",
+  "marinated"
+]
+
+foods = [
+  "salmon",
+  "brussel sprouts",
+  "cauliflower",
+  "spinach salad",
+  "tapioca",
+  "cake",
+  "green beans",
+  "risotto",
+  "beets",
+  "cherries"
+]
+
+print "\nMenu Items of the Day\n\n"
+
+#generate 10 item menu, that randomly combines noun, adjective, food
+
+count = 1
+
+10.times do
```

Comment:
Note that with a `times` loop, you get an iteration variable corresponding to the current count. So this code could be rewritten as:

```ruby
10.times do |n|
  # ..get descrip, how, what...
  print "#{n+1}. #{descrip} #{how} #{what}\n\n"
end
```

Both ways work - use the one that you think makes the most sense to you.